### ISSUE TITLE
CLI command to dump everything

### ISSUE THREAD

**Issue by henn** [CCI-MOC/hil#757]
Once the new CLI is integrated, we should add a command that dumps an entire HIL config.

Something like `hil show --all`.
This way, we can keep the complexity of recursively showing everything out of the API.

**Bonus points:** do something like `hil dump` and `hil restore`, where we can restore a configuration.

**Comment by xuhang57** [comment]
PR merged. Close now and pls reopen if necessary.

**Comment by naved001** [comment]
No, the cli call to dump everything still doesn't exist. What was merged are just API calls to fill in the gaps needed for a call like this.
